.github/workflows/vex.yml

```diff
@@ -0,0 +1,18 @@
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - 'v*.*.*'
+jobs:
+  vexctl:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
+      - name: Set env
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+      - uses: openvex/generate-vex@31b415924ea0d72ed5f2640f1dee59dea6c2770b
+        name: Run vexctl
+        with:
+          product: pkg:golang/k8s.io/minikube@@${{ env.RELEASE_VERSION }}
```

Comment:
@spowelljr Is this the right path?
Or is it  minikube.sigs.k8s.io ?

Reply:
I'm not sure if the `golang` is supposed to be there, but it is `k8s.io/minikube` yes

Reply:
Also just verifying there is supposed to be 2 `@`

Reply:
`k8s.io/minikube` is correct, if you check golang package it comes under `k8s.io/minikube` https://pkg.go.dev/k8s.io/minikube.